.pipelines/pipeline.yaml

```diff
@@ -367,6 +367,14 @@ stages:
       os_version: 'ltsc2022'
       scaleup: 100
 
+  - template: singletenancy/dualstack-overlay/dualstackoverlay-e2e-job-template.yaml
+    parameters:
+      name: "dualstackoverlay_e2e"
+      displayName: AKS DualStack Overlay
+      pipelineBuildImage: "$(BUILD_IMAGE)"
```

Comment:
Aer we using this variable ?

Reply:
I talked to @matmerr, this variable is not required anymore and does not provide any functionality. I have all other occurrences removed in #2077.

Reply:
as discussed, prefer mine PR gets merged first.